<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDerivative</class>
 <widget class="QWidget" name="FormDerivative">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>292</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Численное дифференцирование</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>f(X) =</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditFunction"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X=</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxArgumentValue"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>prec=</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBoxPrec"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>df(x)/dx|x-&gt;-a</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditDerivativeLeft"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>df(x)/dx|x-&gt;+a</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditDerivativeRight"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>df(x)/dx|x-&gt;a</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditDerivativeSymmetrical"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>d2f(x)/dx2 =</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditDerivative2"/>
       </item>
      </layout>
     </item>
     <item row="6" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Кривизна</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditCurvature"/>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Радиус кривизны</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditRadiusOfCurvature"/>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="pushButtonCalculate">
       <property name="text">
        <string>Расчет</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushButtonExit">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
